gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetTest.java

```diff
@@ -219,21 +220,20 @@ public void testPathErrorConsolidator() {
   public void testGenerateCopyEntitiesWhenDestEmpty() throws IOException {
     List<String> expectedPaths = Arrays.asList(METADATA_PATH, MANIFEST_LIST_PATH_0,
         MANIFEST_PATH_0, MANIFEST_DATA_PATH_0A, MANIFEST_DATA_PATH_0B);
-
     MockFileSystemBuilder sourceBuilder = new MockFileSystemBuilder(SRC_FS_URI);
-    sourceBuilder.addPaths(expectedPaths);
+    sourceBuilder.addPathsAndFileStatuses(expectedPaths, false);
```

Comment:
`addPathsAndFileStatuses()` doesn't make sense when called w/ a collection of paths and a boolean.  let's think from the perspective of a new maintaner.  anyway, seems `.addPaths(ps)` has equivalent semantics and inserts `false` behind the scenes, no?